front/src/common/authorization/hooks/useCheckUserRole.ts

```diff
@@ -0,0 +1,14 @@
+const useCheckUserRole = (requiredRoles: string[] = []) => {
+  if (requiredRoles.length === 0) {
+    return true;
+  }
+
+  // TODO AUTH:
+  // - get user role when it is implemented
+  // - uncomment when user roles are implemented
+  // const userRoles: string[] = [];
+  // return requiredRoles.some((role) => userRoles.includes(role))
```

Comment:
Should probably be `every()` instead of `some()` here?

Reply:
All the checks for permissions and roles are not fully ready yet. I propose to keep it as it for now. Once the backend implements this, we can properly implement it on the front end as well.

Reply:
Maybe we can remove this commented code (lines 9-10) ?